src/app/viewChooser/ViewChooserFilter.tsx

```diff
@@ -20,7 +20,7 @@ export function ViewChooserFilter(props: IViewChooserFilterProps) {
 
   return (
     <div className="view-filter input-group flex-nowrap">
-      <input className="form-control border-end-0" type="search" placeholder="Search" value={filter} onChange={(evt) => setFilter(evt.target.value)} />
+      <input className="form-control border-end-0" type="search" placeholder="Search Views" value={filter} onChange={(evt) => setFilter(evt.target.value)} />
```

Comment:
```suggestion
      <input className="form-control border-end-0" type="search" placeholder="Search views" value={filter} onChange={(evt) => setFilter(evt.target.value)} />
```

Use sentence case according to style guide.